prinz/src/main/scala/pl/touk/nussknacker/prinz/mlflow/repository/MLFBucketRepository.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.prinz.mlflow.repository
+
+import java.io.InputStream
+
+import pl.touk.nussknacker.prinz.mlflow.MlflowConstants.{S3_ACCESS_KEY, S3_SECRET_KEY, S3_URL}
+import pl.touk.nussknacker.prinz.util.amazon.S3Client
+
+class MLFBucketRepository(val bucketName: String) {
+
+  private val s3Client = S3Client(S3_URL, S3_ACCESS_KEY, S3_SECRET_KEY)
+
+  def getMLModelFile(experimentId: Int, runId: String): InputStream = s3Client
+    .downloadFile(bucketName, s"$experimentId/$runId/artifacts/model/MLmodel")
+}
+
+object MLFBucketRepository {
```

Comment:
Same here.

Reply:
in my opinion we have to use it (or even full namy if this is ugly - it doesn't const anything but can make code less buggy)